src/wp-includes/media.php

```diff
@@ -1050,9 +1050,18 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 			'decoding' => 'async',
 		);
 
+		/**
+		 * Filters the context in which wp_get_attachment_image() is used.
+		 *
+		 * @since 6.3.0
+		 *
+		 * @param string $context The context. Default 'wp_get_attachment_image'.
+		 */
+		$context = apply_filters( 'wp_get_attachment_image_context', 'wp_get_attachment_image' );
```

Comment:
```suggestion
		$context = apply_filters( 'wp_get_attachment_image_context', __FUNCTION__ );
```

Reply:
While `wp_get_attachment_image` is also the function name, it's safer to rely on a string. For example, if this filter was moved to a sub function called by `wp_get_attachment_image()`, the way the code is currently written it would still work, but if we used `__FUNCTION__`, it would break.

Reply:
If we move the filter the future? What? When would that happen?

Reply:
I'm not saying we'll move it, but that example is a reason why using `__FUNCTION__` here would be more error-prone.